lib/views/login/login_view.dart

```diff
@@ -0,0 +1,174 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+import 'package:kayla_flutter_ic/gen/assets.gen.dart';
+import 'package:kayla_flutter_ic/views/login/login_form.dart';
+
+class LoginView extends StatefulWidget {
+  const LoginView({super.key});
+
+  @override
+  State<LoginView> createState() => _LoginViewState();
+}
+
+class _LoginViewState extends State<LoginView> with TickerProviderStateMixin {
+  late AnimationController _logoAnimationController;
+  late Animation<double> _logoAnimation;
+  bool _isLogoAnimated = false;
+
+  late AnimationController _backgroundAnimationController;
+  late Animation<double> _backgroundAnimation;
+
+  late AnimationController _formAnimationController;
+  late Animation<double> _formAnimation;
+
+  SizedBox get _background => SizedBox(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
+          children: [
+            _backgroundImage,
+            _animatedBackground,
+          ],
+        ),
+      );
+
+  Image get _backgroundImage => Image(
+        image: Assets.images.nimbleBackground.image().image,
+        fit: BoxFit.cover,
+        alignment: Alignment.center,
+      );
+
+  FadeTransition get _animatedBackground => FadeTransition(
+        opacity: _backgroundAnimation,
+        child: _linearGradientBlurBackground,
+      );
+
+  Stack get _linearGradientBlurBackground => Stack(
+        children: [
+          _backgroundImage,
+          ImageFiltered(
+            imageFilter: ImageFilter.blur(sigmaX: 10, sigmaY: 10),
+            child: ShaderMask(
+              shaderCallback: (rectangle) {
+                return LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  colors: [
+                    Colors.black.withOpacity(0.2),
+                    Colors.black.withOpacity(1),
+                  ],
+                ).createShader(rectangle);
+              },
+              blendMode: BlendMode.overlay,
+              child: _backgroundImage,
+            ),
+          ),
+        ],
+      );
+
+  AnimatedPositioned get _animatedLogo => AnimatedPositioned(
+        duration: const Duration(seconds: 1),
```

Comment:
@nmint8m We use this const at many places, so it would be better if we define it as a local constant variable. In the future, if we want to change the animation duration, we only need to change in one place. What do you think? 🤔

Reply:
I will define it as a global constant so that I can reuse many places without accidentally changing them.